src/components/UpdateWhitelistUsersButton/UpdateModal/styles.less

```diff
@@ -0,0 +1,63 @@
+.update-whitelist-users {
+  .download-template {
+    margin-left: 8px;
+  }
+  .update-way-radio-wrapper {
+    height: 40px;
+    padding: 1px;
+    border: 1px solid #070a26;
+    border-radius: 8px;
+    .radio-item {
+      flex: 1;
+      &:first-child {
+        border-radius: 6px 0 0 6px;
+      }
+      &:last-child {
+        border-radius: 0 6px 6px 0;
+      }
+      &.radio-item-active {
+        background-color: #070a26;
```

Comment:
.

Reply:
It will be improved in the next version.